ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/PrivacySendTransaction.java

```diff
@@ -33,23 +31,20 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public abstract class PrivacySendTransaction extends PrivacyApiMethod {
+public class PrivacySendTransaction {
 
   private static final Logger LOG = LogManager.getLogger();
 
-  protected final PrivateTransactionHandler privateTransactionHandler;
+  protected final PrivacyController privacyController;
   protected final TransactionPool transactionPool;
 
   public PrivacySendTransaction(
-      final PrivacyParameters privacyParameters,
-      final PrivateTransactionHandler privateTransactionHandler,
-      final TransactionPool transactionPool) {
-    super(privacyParameters);
-    this.privateTransactionHandler = privateTransactionHandler;
+      final PrivacyController privacyController, final TransactionPool transactionPool) {
+    this.privacyController = privacyController;
     this.transactionPool = transactionPool;
   }
 
-  protected PrivateTransaction validateAndDecodeRequest(final JsonRpcRequestContext request)
+  public PrivateTransaction validateAndDecodeRequest(final JsonRpcRequestContext request)
```

Comment:
nit: does this need to be public? Seems like too much visibility (without looking further)